src/quo2/components/list_items/saved_contact_address/view.cljs

```diff
@@ -0,0 +1,94 @@
+(ns quo2.components.list-items.saved-contact-address.view
+  (:require [quo2.components.avatars.user-avatar.view :as user-avatar]
+            [quo2.components.avatars.account-avatar.view :as account-avatar]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [quo2.components.list-items.saved-contact-address.style :as style]
+            [reagent.core :as reagent]
+            [quo2.components.icon :as icon]
+            [utils.i18n :as i18n]))
+
+(defn- f-internal-view
+  []
+  (let [state       (reagent/atom :default)
```

Comment:
I think we need to expose the `state` prop to control the component from the parent. Maybe, just for the `active` state. WYDT?

Reply:
Hmm I think we would not want to expose the whole state atom, but I could add a boolean prop to set if we want to set active state? WDYT?

Reply:
Oh, no. Not the atom. I meant the `state` prop (Figma 1-1).

Most likely we might need a prop for the `selected` state. Right now, the component handles it but we need to set it from the screen which uses this component.